app/components/troubleshooting_options_component.rb

```diff
@@ -2,14 +2,19 @@ class TroubleshootingOptionsComponent < BaseComponent
   renders_one :header, 'TroubleshootingOptionsHeadingComponent'
   renders_many :options, BlockLinkComponent
 
-  attr_reader :tag_options
+  attr_reader :options_from_constructor, :tag_options
```

Comment:
looks like we never read `options_from_constructor` outside of this class, so maybe make that private?

Reply:
Yeah, we could 👍

Reply:
I just got rid of the attribute altogether in favor of consolidating to the single `options` method as part of 7000581ef.